promotion/datastore.go

```diff
@@ -691,3 +703,80 @@ limit 1`
 
 	return attempted, nil
 }
+
+// This code can be deleted once https://github.com/brave-intl/bat-go/issues/263 is addressed.
+
+// GetOrder queries the database and returns an order
+func (pg *Postgres) GetOrder(orderID uuid.UUID) (*Order, error) {
+	statement := "SELECT * FROM orders WHERE id = $1"
```

Comment:
Would it make sense to put all these SQL statements as constants so we are not instantiating them every time we call these functions?

Reply:
i was looking into prepared statements, and think that might be an even better way to go, where we can use it. https://jmoiron.github.io/sqlx/#preparedStmts

Reply:
Since this is used pretty much everywhere I've created an issue to address it. https://github.com/brave-intl/bat-go/issues/275